src/components/timecard/edit-shift-timecard/EditShiftTimecard.jsx

```diff
@@ -68,9 +69,18 @@ const EditShiftTimecard = ({
             className="govuk-summary-list__key govuk-!-width-two-thirds"
             style={{ paddingBottom: '20px', paddingTop: '10px' }}
           >
-            Shift
+            Time period
           </dt>
-          <dd className="govuk-summary-list__value"></dd>
+          <dd
+            className="govuk-summary-list__value"
+            style={{ whiteSpace: 'nowrap' }}
+          >
+            {timeEntryExists
+              ? timecardDate === formatDate(timeEntry.startTime)
+                ? 'Shift'
+                : 'Shift (continued)'
```

Comment:
MIght be worth using the getting the 'Shift' text from the map returned by the `getTimePeriodTypesMap` function as that comes from the API. Easier maintenance.

Reply:
a change has been made...it's caused 17 tests to fail, but is this change you meant?

Reply:
This will be revisited as a sub-task after the ui revamp

Reply:
Yeah that was the change, resolving after discussion and should be picked up as part of the revamp.